formspree/forms/views.py

```diff
@@ -72,6 +75,8 @@ def send(email_or_string):
             )
         ), 500
 
+    # fail if the host (referrer) is empty, or it's google
+    # (IE form was hosted on google, via AMP or something?)
     if not host or host == 'www.google.com':
```

Comment:
This was actually created before we implemented reCAPTCHA because of spammers. I don't think we'll need this anymore.

Reply:
Yes, I think we don't need it. It was a temporary antispam measure.